src/coreclr/jit/lsra.h

```diff
@@ -735,7 +735,7 @@ class LinearScan : public LinearScanInterface
     // Hence the "SmallFPSet" has 5 elements.
     CLANG_FORMAT_COMMENT_ANCHOR;
 
-#if defined(TARGET_AMD64)
+#if defined(TARGET_AMD64) || defined(TARGET_WASM)
```

Comment:
There changes should not be needed now that the whole file is ifdefed out

Reply:
Removed.